doc/search-quality-metrics.md

```diff
@@ -1,11 +1,34 @@
-# Rank evaluation
+# Search Quality Metrics
 
-One way we assess the affect of search-api changes to relevancy, is by using
-Elasticsearch's [Ranking Evaluation API](ranking_evaluation_api).
+We assess the quality of Search API search results with two metrics:
+Click Through Rate (an online metric) and Normalised Discounted Cumulative Gain
+([nDCG][ndcg]).
```

Comment:
This link doesn't work because the definition below is for `ncdg`.  I'd prefer fixing that to making this link use the same typo.